lib/components/user/monitored-trip/saved-trip-editor.js

```diff
@@ -22,31 +23,37 @@ const SavedTripEditor = props => {
       {
         pane: panes.basics,
         props,
-        title: 'Trip information'
+        title: <FormattedMessage id='components.SavedTripEditor.tripInformation' />
       },
       {
         pane: panes.notifications,
         props,
-        title: 'Trip notifications'
+        title: <FormattedMessage id='components.SavedTripEditor.tripNotifications' />
       }
     ]
 
     return (
-      <div>
+      <>
         <BackLink />
         <StackedPaneDisplay
           onCancel={onCancel}
           paneSequence={paneSequence}
-          title={isCreating ? 'Save new trip' : 'Edit saved trip'}
+          title={<FormattedMessage
+            id={`components.SavedTripEditor.${isCreating ? 'saveNew' : 'editSaved'}Trip`}
```

Comment:
Please do not create dynamic message IDs as it is hard to automatically keep track of which IDs are in the code. Instead, please create a separate variable for each possible message.